From d8e8ab7a56ca2020500504b50eddcf7c07cf528a Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Tue, 22 Sep 2020 12:24:14 -0700
Subject: [PATCH 1638/1851] v4l: xilinx: vpss-csc: Fix warning for UNINIT

commit 011898d2c2498ede863fcb163d2c5c7bafeaea31 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix the warnings for the UNINIT type.

The actual warnings are:
For xcsc_parse_of():
Event var_decl: Declaring variable "video_width" without initializer.
Event uninit_use: 	Using uninitialized value "video_width[0]".

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-vpss-csc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-vpss-csc.c b/drivers/media/platform/xilinx/xilinx-vpss-csc.c
index 752e7b4f047b..8e6fd16e5630 100644
--- a/drivers/media/platform/xilinx/xilinx-vpss-csc.c
+++ b/drivers/media/platform/xilinx/xilinx-vpss-csc.c
@@ -984,7 +984,7 @@ static int xcsc_parse_of(struct xcsc_dev *xcsc)
 	struct device_node *ports, *port;
 	int rval;
 	u32 port_id = 0;
-	u32 video_width[2];
+	u32 video_width[2] = {0};
 
 	rval = of_property_read_u32(node, "xlnx,max-height", &xcsc->max_height);
 	if (rval < 0) {
-- 
2.31.1

